validateBuffer has non-prefixed, unused arguments

## Sorting

- **I'm submitting a ...**

  - [x] bug report
  - [ ] feature request
  - [ ] support request

## Expected Behavior

<!--- If this is related to a TypeScript issue, then include the way that you expected to be able to write the code  -->

current definition
file: packages/runtime/src/routeGeneration/templateHelpers.ts
line: 457
```ts
  public validateBuffer(name: string, value: string) {
    return Buffer.from(value);
  }
```

## Current Behavior
for a `tsconfig.json` settings with `
```json
{
  "compilerOptions": {
    "noUnusedParameters": true
  }
}
```
ive got error:
```
node_modules/@tsoa/runtime/src/routeGeneration/templateHelpers.ts:457:25 - error TS6133: 'name' is declared but its value is never read.

457   public validateBuffer(name: string, value: string) {
                            ~~~~


Found 1 error.
```

## Possible Solution
```ts
  public validateBuffer(_name: string, value: string) {
    return Buffer.from(value);
  }
```

## Context (Environment)

Version of the library: 3.8.0
Version of NodeJS: v16.4.2
Version of Typescript: 4.3.5

- Confirm you were using yarn not npm: [x]

--- a/packages/runtime/src/routeGeneration/templateHelpers.ts
+++ b/packages/runtime/src/routeGeneration/templateHelpers.ts
@@ -475,7 +475,7 @@ export class ValidationService {
     return arrayValue;
   }
 
-  public validateBuffer(name: string, value: string) {
+  public validateBuffer(_name: string, value: string) {
     return Buffer.from(value);
   }
 
@@ -496,14 +496,7 @@ export class ValidationService {
       // Clean value if it's not undefined or use undefined directly if it's undefined.
       // Value can be undefined if undefined is allowed datatype of the union
       const validateableValue = value ? JSON.parse(JSON.stringify(value)) : value;
-      const cleanValue = this.ValidateParam(
-        { ...subSchema, validators: { ...property.validators, ...subSchema.validators } },
-        validateableValue,
-        name,
-        subFieldError,
-        parent,
-        swaggerConfig,
-      );
+      const cleanValue = this.ValidateParam({ ...subSchema, validators: { ...property.validators, ...subSchema.validators } }, validateableValue, name, subFieldError, parent, swaggerConfig);
       subFieldErrors.push(subFieldError);
 
       if (Object.keys(subFieldError).length === 0) {

